database/schema/01-auth.sql

```diff
@@ -1,4 +1,14 @@
 CREATE TABLE account
 (
     address TEXT NOT NULL PRIMARY KEY
+);
+
+
+CREATE TABLE vesting_account
+(
+    address             TEXT        NOT NULL REFERENCES account (address) PRIMARY KEY,
+    original_vesting    JSONB       NOT NULL DEFAULT '{}',
+    end_time            TEXT        NOT NULL,
+    start_time          TEXT        NOT NULL,
+    vesting_periods     JSONB       NOT NULL DEFAULT '{}'
```

Comment:
I'm not a big fan of having a `JSONB` to represent a list of vesting periods since it's not possible to execute proper queries on this kind of type. Instead, I think we should create another table named `vesting_period` and insert those periods there. This would allow to query all the periods ending on a specific date or other things which might be useful for X or BigDipper